api/src/users/users.module.ts

```diff
@@ -28,6 +29,7 @@ import { ImageDao } from '../image/daos/image.dao';
 		AuthModule,
 		ExportModule,
 		ImageModule,
+		MoviesModule,
```

Comment:
Why do we need this here?

Reply:
I'm not sure why, but it won't compile otherwise